style.css

```diff
@@ -260,13 +250,131 @@
     flex-grow: 0;
   }
 
+  .project-modal {
+    display: none;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    position: fixed;
+    z-index: 1;
+    padding-top: 40px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  .project-action-buttons {
+    display: flex;
+    padding: 4px 9px;
+    justify-content: space-between;
+    align-items: center;
+    gap: 12px;
+    color: #6070ff;
+    width: 95%;
+  }
+
+  .view-project-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    border-radius: 8px;
+    padding: 5px;
+    border: 1px solid #6070ff;
+    width: 136px;
+    height: 48px;
+  }
+
+  .view-project-btn > p {
+    color: #396df2;
+    text-align: center;
+    font-size: 15px;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    line-height: 20px;
+  }
+
+  .modal-content {
+    margin-left: 2%;
+    margin-right: 2%;
+    background-color: #fff;
+    border-radius: 8px;
+    padding: 8px;
+    padding-bottom: 60px;
+  }
```

Comment:
- [ ] Please always try to make your design as close as possible to provided design, if you look at the below-attached table of design comparison, your design looks unprofessional and not consistent according to screen height. Your pop is actually going outside the viewport and it shouldn't happen. Come to your desktop popUp design which looks disastrous and pretty bad to the eyes 👁️. I would request you kindly fix your PopUp issue, and make it looks professional and similar to Figma's design.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-31-56](https://github.com/NoelLincoln/mportfolio/assets/66476329/8b39c374-0973-4b4e-ae70-70558d910358)

</td>
<td>

![Screenshot from 2023-06-30 21-31-36](https://github.com/NoelLincoln/mportfolio/assets/66476329/3cbf8564-b2ca-4d0c-950e-03e3cf064036)

</td>
</tr>
</table>

### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-39-57](https://github.com/NoelLincoln/mportfolio/assets/66476329/eaed1c3c-b6ae-4536-9437-9f72f2d0eb8b)

</td>
<td>

![Screenshot from 2023-06-30 21-39-31](https://github.com/NoelLincoln/mportfolio/assets/66476329/ab5d00e1-60d0-492f-9c0e-c3a7db6af7c5)

</td>
</tr>
</table>